packages/material-ui/src/Tooltip/Tooltip.js

```diff
@@ -402,6 +408,28 @@ const Tooltip = React.forwardRef(function Tooltip(props, ref) {
     }, leaveTouchDelay);
   };
 
+  React.useEffect(() => {
+    if (!open) {
+      return undefined;
+    }
+
+    /**
+     * @param {KeyboardEvent} nativeEvent
+     */
+    function handleKeyDown(nativeEvent) {
```

Comment:
arrow function as in the other places?

Reply:
> (for nested scopes)

What does this mean?

Reply:
I thought we were following named function for top-level scope and arrow function for nested levels, following pros and cons of https://stackoverflow.com/questions/34361379/are-arrow-functions-and-functions-equivalent-exchangeable

```suggestion
    const handleKeyDown = (nativeEvent) =>
```

Reply:
Does it matter? They end up as `function` anyway.